### DISCUSSION TITLE
Help with setting breakpoints, not just points

### DISCUSSION THREAD

**Discussion by Katah1996** [osmandapp/OsmAnd#20790] [Q&A]
We use OsmAnd as a route navigator, that is, we download a .gpx file from our application, or from the Rinkai web portal and import it into OsmAnd and get a track of the route we need to drive, but unfortunately we get a route with stop points, but they are simply displayed as points, on the map there are points with stars, but the map does not recognize them as stops, could you tell me how to fix it, can send you a screenshot or part of the code.

**Comment by vshcherb** [comment]
If you have - https://docs.osmand.net/docs/user/map/tracks/#types-of-tracks - Track of **Route type** points should be handled automatically. Waypoints are not related to navigation but simply points to probably visit / see on the map
